graphql/resolvers.go

```diff
@@ -558,6 +558,43 @@ func (r *queryResolver) TaskLogs(ctx context.Context, taskID string) (*RecentTas
 	return &RecentTaskLogs{EventLogs: apiEventLogPointers, TaskLogs: taskLogPointers, AgentLogs: agentLogPointers, SystemLogs: systemLogPointers}, nil
 }
 
+func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string) ([]*PatchBuildVariant, error) {
+	patch, err := r.sc.FindPatchById(patchID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding patch `%s`: %s", patchID, err))
+	}
+
+	var tasksByVariant map[string][]*PatchBuildVariantTask = map[string][]*PatchBuildVariantTask{}
+	for _, variant := range patch.Variants {
+		tasksByVariant[*variant] = []*PatchBuildVariantTask{}
+	}
+	tasks, err := r.sc.FindTasksByVersion(patchID, task.DisplayNameKey, []string{}, 1, 0, 1000)
```

Comment:
all the patch's tasks need to be fetched for this query -- 1000 is an arbitrary number that i assume will cover all of them. should it be higher?